lib/inherited_proofing/mhv/configuration.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+
+module InheritedProofing
+  module MHV
+    class Configuration < Common::Client::Configuration::REST
+      def valid_id_url
+        Settings.mhv.inherited_proofing.valid_id_url
+      end
+
+      def vacct_info_url
+        Settings.mhv.inherited_proofing.vacct_info_url
```

Comment:
why don't we have these urls just the endpoints, and build from the base url, that way we only have to update a single field in the settings for different stacks?